test/StandardToken.js

```diff
@@ -5,8 +5,13 @@ var StandardTokenMock = artifacts.require('./helpers/StandardTokenMock.sol');
 
 contract('StandardToken', function(accounts) {
 
+  let token;
```

Comment:
This refactoring might fail if the developer has previously ran tests that involve transactions within the testrpc environment. Because can ran out of funds before getting to complete all the test cases.
Not sure if the refactoring is worth it.